plugins/GSdx/GSSettingsDlg.cpp

```diff
@@ -767,6 +767,11 @@ bool GSHacksDlg::OnMessage(UINT message, WPARAM wParam, LPARAM lParam)
 
 			EndDialog(m_hWnd, id);
 		} break;
+
+		case IDCANCEL:
+		{
+			EndDialog(m_hWnd, IDCANCEL);
+		} break;
```

Comment:
All the cases end with a break. I followed the example of the cancel case of the shader dialog on line 593.